initializers/rack_groups.yml

```diff
@@ -0,0 +1,3 @@
+#- name: cage 101
+#  slug: cage-101
+#  site: SING 1
```

Comment:
Since I had to request a change anyway:
All other files are commented slightly different.
It's just a space, but it seems to matter in an editor like VS Code.
When I un-commented your file with `CMD+/`, it was malformatted:

```
- name: cage 101
 slug: cage-101
 site: SING 1
```

But if there is an additional space, like as follows so, everything is fine.

```
# - name: cage 101
#   slug: cage-101
#   site: SING 1
```

I know this sounds like I'm being pedantic.
But I would like to have the format consistent, because I would like to add a new automated test in the future.
This test would remove all the `# ` prefixes and try to start up Netbox Docker (i.e. `docker-compose up`). That would cause the build system to complain (instead of me ;) ) if there are inconsistencies in the initializer samples. But if one file is formatted slightly different than all the others, that test could not work.

Reply:
I have added an additional space.
Looking forward to the automated test!